javascript2/week1/homework.md

```diff
@@ -1,3 +1,5 @@
+<table><tr><td>⚠️ You do not need to complete these homeworks. We are currently trialling alternative homeworks for JS2 and JS3. More info <a href="https://github.com/HackYourFuture-CPH/js-projects">here.</a></td></tr></table>
```

Comment:
I like it 🏗️ 

Some ideas:
- I have a feeling the note is easy to miss due to "banner blindness", maybe we can add a slight background color or colored line to catch attention? Like the warning section on [this page](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository) or the archived banner on repositories
![grafik](https://github.com/HackYourFuture-CPH/JavaScript/assets/4127167/dbce7a04-5dd7-47c7-857f-1678421a85ba)
![grafik](https://github.com/HackYourFuture-CPH/JavaScript/assets/4127167/83051570-3b82-493e-a802-2a914a07bb4e)
- This may be contentious, but I'm inclined to suggest simplifying language for this warning, especially since quite a few trainees are still beginners in English. In this context, I'd suggest to replace "trialling" with "testing" or similar.

Reply:
Great suggestions. TIL that markdown has support for those type of warnings :D Thanks! Updating now...